app/controllers/v0/sessions_controller.rb

```diff
@@ -70,7 +70,7 @@ def new
     # authn_context is the policy, connect represents the ID.me flow
     # TODO: DEPRECATED
     def multifactor
-      render json: { multifactor_url: SAML::SettingsService.mfa_url(current_user) }
+      render json: { multifactor_url: SAML::SettingsService.mfa_url(current_user, alt_relay: alternate_saml_relay?) }
```

Comment:
This might be the right opportunity to remove this deprecated route. It is no longer being used by the front end.